data/uking_functions.csv

```diff
@@ -94369,8 +94369,8 @@ Address,Quality,Size,Name
 0x0000007101266ecc,U,000060,
 0x0000007101266f08,U,000100,
```

Comment:
These two functions might be the DebugInput::SingletonDisposer destructor functions (which you get for free when you use the SINGLETON macro). 

You can use `tools/listsym DebugInput` to figure out their names and then list them

Reply:
```
% ./tools/check -mw _ZN4ksys10DebugInput18SingletonDisposer_D0Ev
ksys::DebugInput::SingletonDisposer_::~SingletonDisposer_() (_ZN4ksys10DebugInput18SingletonDisposer_D0Ev)
OK
note: changing status from NotDecompiled to Matching

% ./tools/check -mw _ZN4ksys10DebugInput18SingletonDisposer_D1Ev
ksys::DebugInput::SingletonDisposer_::~SingletonDisposer_() (_ZN4ksys10DebugInput18SingletonDisposer_D1Ev)
OK
```

```
0x0000007101266ecc,U,000060,
0x0000007101266f08,O,000100,_ZN4ksys10DebugInput18SingletonDisposer_D1Ev
0x0000007101266f6c,O,000108,_ZN4ksys10DebugInput18SingletonDisposer_D0Ev
0x0000007101266fd8,O,000152,_ZN4ksys10DebugInput14createInstanceEPN4sead4HeapE
0x0000007101267070,O,000160,_ZN4ksys10DebugInput6updateEv
```